packages/lit-dev-content/site/tutorials/content/reactivity/04.md

```diff
@@ -0,0 +1,37 @@
+One way your component can react to property changes is to compute values based on the new and existing property values. 
+
+The `willUpdate()` callback is the ideal place to do these calculations in Lit. It's called near the beginning up the update cycle, before the component renders, so any properties you calculate during `willUpdate()` are available in your `render()` method.
+
+The `willUpdate()` method receives a map of changed properties as an argument, where the keys are property names and the values are the old values for each changed property. In this case, we'll compute a hash value based on the `name` property.
```

Comment:
2 paragraphs back you call it a callback. Or maybe you want them both to be method idk

```suggestion
The `willUpdate()` callback receives a map of changed properties as an argument, where the keys are property names and the values are the old values for each changed property. In this case, we'll compute a hash value based on the `name` property.
```